addons/l10n_ar_website_sale/views/templates.xml

```diff
@@ -43,7 +43,10 @@
 
     </template>
 
-    <template id="address_b2b" inherit_id="website_sale.address_b2b">
+    <template id="address" inherit_id="website_sale.address">
+        <xpath expr="//t[contains(@t-if, 'website_sale.address_b2b')]" position="attributes">
+            <attribute name="t-if" separator=" " add="or res_company.country_id.code == 'AR'"/>
```

Comment:
I find it nice that this is possible (til add operator), but it makes it a bit weird if we have multiple countries like this, although it would work.  The xpath might also not be the most robust in case someone does other customizations.  Maybe we can put another method in the Python than the is_view_active and inherit that one?  @nboulif @Williambraecky @rdeodoo

Reply:
it feels weird if every module (and every l10n on top of that) is going to add its own diff in the regular code base (in this case website_sale) to fit its needs.
I proposed this solution to @nboulif as it does not change the code of website_sale (except that the customize_show view is inlined).
I'll let @Williambraecky take over this one as it hits master, unless he wants me to handle it :+1: 
So wait for his opinion

Reply:
I am fine with with it for now, if we are to add more of the same override in the future I would rather have a function that we may call instead but for this single occurence it's ok for me.